vsphere/resource_vsphere_compute_cluster.go

```diff
@@ -577,6 +578,51 @@ func resourceVSphereComputeCluster() *schema.Resource {
 					},
 				},
 			},
+			"vsan_stretched_cluster_enabled": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
+				Description: "Whether the vSAN stretched cluster is configured for the cluster.",
+			},
+			"vsan_stretched_cluster_conf": {
+				Type:        schema.TypeSet,
```

Comment:
It seems like lower down you are expecting a single entry in the Set. This is a common practice for creating that nice "nested block" of config for the user. We typically do `schema.TypeList` and `MaxItems: 1` to create that. Which will allow you to not have to convert around the TypeSet in the rest of the implementation.

Reply:
Yes, we only need a nested block of config. Could you please help provide an example of using the schema.TypeList, so that we could follow the current coding specifications?

Reply:
Example in the provider code :
https://github.com/hashicorp/terraform-provider-vsphere/blob/main/vsphere/resource_vsphere_vnic.go#L235

HCL example from the provider documentation

```
resource "vsphere_vnic" "v1" {
  host                    = data.vsphere_host.h1.id
  distributed_switch_port = vsphere_distributed_virtual_switch.d1.id
  distributed_port_group  = vsphere_distributed_port_group.p1.id
  ipv4 {
    dhcp = true
  }
  netstack = "vmotion"
}
```